--- soundtracker-0.6.7/app/Makefile.am.orig	2003-05-18 12:20:55.000000000 +0200
+++ soundtracker-0.6.7/app/Makefile.am	2004-01-18 01:58:47.546089168 +0100
@@ -57,30 +57,6 @@
 
 soundtracker_LDADD = drivers/libdrivers.a mixers/libmixers.a ${ST_S_JACK_LIBS}
 
-install-exec-local:
-	case `uname` in \
-	  OpenBSD) \
-	    chown root.wheel $(bindir)/soundtracker; \
-	    ;; \
-	  NetBSD) \
-	    chown root.wheel $(bindir)/soundtracker; \
-	    ;; \
-	  Darwin) \
-	    chown root:admin $(bindir)/soundtracker; \
-	    ;; \
-	  *) \
-	    chown root.root $(bindir)/soundtracker; \
-	    ;; \
-	esac
-	chmod +s $(bindir)/soundtracker
-	@echo ""
-	@echo "***"
-	@echo "*** Installing SoundTracker suid root."
-	@echo "*** This improves sound responsivity, but may be a security risk"
-	@echo "*** if this is a multi-user machine. See file INSTALL."
-	@echo "***"
-	@echo ""
-
 
 stdir = $(datadir)/soundtracker
 
