--- soundtracker-0.6.8/app/Makefile.am.orig	2006-02-25 13:30:54.000000000 +0100
+++ soundtracker-0.6.8/app/Makefile.am	2006-10-23 22:31:11.417543750 +0200
@@ -58,31 +58,6 @@
 
 soundtracker_LDADD = drivers/libdrivers.a mixers/libmixers.a ${ST_S_JACK_LIBS}
 
-install-exec-local:
-	case `uname` in \
-	  OpenBSD) \
-	    chown root.wheel $(bindir)/soundtracker; \
-	    ;; \
-	  NetBSD) \
-	    chown root.wheel $(bindir)/soundtracker; \
-	    ;; \
-	  Darwin) \
-	    chown root:admin $(bindir)/soundtracker; \
-	    ;; \
-	  *) \
-	    chown root:root $(bindir)/soundtracker; \
-	    ;; \
-	esac
-	chmod +s $(bindir)/soundtracker
-	@echo ""
-	@echo "***"
-	@echo "*** Installing SoundTracker suid root."
-	@echo "*** This improves sound responsivity, but may be a security risk"
-	@echo "*** if this is a multi-user machine. See file INSTALL."
-	@echo "***"
-	@echo ""
-
-
 stdir = $(datadir)/soundtracker
 
 #INCLUDES = -DDATADIR=\"$(stdir)\" \
