--- soundtracker-0.6.6/configure.in.orig	Wed Aug 14 13:34:57 2002
+++ soundtracker-0.6.6/configure.in	Wed Aug 14 13:38:40 2002
@@ -32,6 +32,7 @@
   AM_CONDITIONAL(GASP_IS_THERE, test -n "$GASPPATH")
 else
   AC_DEFINE(NO_ASM)
+  AM_CONDITIONAL(GASP_IS_THERE, false)
 fi
 
 CFLAGS="$CFLAGS -Wall"
@@ -169,10 +170,15 @@
   fi
   AC_CHECK_LIB(asound,snd_pcm_capture_params,
 	[AM_CONDITIONAL(DRIVER_ALSA, test x$alsa_support = xyes)
-	AC_DEFINE(DRIVER_ALSA)])
+	AC_DEFINE(DRIVER_ALSA)],
+	[AM_CONDITIONAL(DRIVER_ALSA, false)])
   AC_CHECK_LIB(asound,snd_pcm_channel_params,
 	[AM_CONDITIONAL(DRIVER_ALSA_050, test x$alsa_support = xyes)
-	AC_DEFINE(DRIVER_ALSA_050)])
+	AC_DEFINE(DRIVER_ALSA_050)],
+	[AM_CONDITIONAL(DRIVER_ALSA_050, false)])
+else
+  AM_CONDITIONAL(DRIVER_ALSA, false)
+  AM_CONDITIONAL(DRIVER_ALSA_050, false)
 fi                                   
 
 dnl -----------------------------------------------------------------------
@@ -192,6 +198,8 @@
     CFLAGS="$CFLAGS $ESD_CFLAGS"
     LIBS="$LIBS $ESD_LIBS"
   fi
+else
+  AM_CONDITIONAL(DRIVER_ESD, false)
 fi
 
 dnl -----------------------------------------------------------------------
